bin/data-push-flow.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+const fs = require('fs')
+const path = require('path')
+
+const minimist = require('minimist')
+const urljoin = require('url-join')
+const {Dataset} = require('data.js')
+const { write: copyToClipboard } = require('clipboardy')
+
+// Ours
+const config = require('../lib/utils/config')
+const {customMarked} = require('../lib/utils/tools.js')
+const {handleError} = require('../lib/utils/error')
+const wait = require('../lib/utils/output/wait')
+const {DataHub} = require('../lib/utils/datahub.js')
+const {authenticate} = require('../lib/login')
+const info = require('../lib/utils/output/info.js')
+
+
+const argv = minimist(process.argv.slice(2), {
+  string: ['push-flow'],
+  boolean: ['help', 'debug', 'interactive'],
+  alias: {help: 'h', interactive: 'i'}
+})
+
+const pushMarkdown = fs.reSync(path.join(__dirname, '../docs/push-flow.md'), 'utf8')
+const help = () => {
+  console.log('\n' + customMarked(pushMarkdown))
+}
+
+if (argv.help) {
+  help()
+  process.exit(0)
+}
+
+Promise.resolve().then(async () => {
+  let stopSpinner = () => {}
+  // First check if user is authenticated
+  const apiUrl = config.get('api')
+  const token = config.get('token')
+  let out
+  try {
+    out = await authenticate(apiUrl, token)
+  } catch (err) {
+    handleError(err)
+    process.exit(1)
+  }
+  if (!out.authenticated) {
+    info('You need to login in order to push your data. Please, use `data login` command.')
+    process.exit(0)
+  }
+  try {
+    const filePath = argv._[0] || process.cwd()
```

Comment:
I think you copied this code from another command, but this is not a `filePath` anymore. I think it is now just `path_` or `datasetPath`